adoc/Private-Registry-Installation.adoc

```diff
@@ -589,108 +589,58 @@ kubectl create namespace redis-operator
 
-... Create a secret holding the redis password:
+... Configure `RedisFailover` object:
 +
-[source,bash]
-----
-kubectl -n registry create secret generic redis-auth --from-literal=password="<password-value>"
-----
-
-... Install `RedisFailover` object:
-+
-The Redis HA configuration needs to be specified as a `RedisFailover` {kube} CRD object.
+The Redis HA configuration needs to be specified in the `redisfailover` section of harbor-values.yaml.
 The following is an example configuration:
 +
+.harbor-values.yaml
 [source,yaml]
 ----
-apiVersion: databases.spotahome.com/v1
-kind: RedisFailover
-metadata:
-  name: harbor-redis
-spec:
-  sentinel:
-    replicas: 3
-    image: registry.suse.com/harbor/harbor-redis:2.1.1
-    securityContext:
-      runAsUser: 999
-      runAsGroup: 999
-      fsGroup: 999
-    resources:
-      requests:
-        cpu: 100m
-      limits:
-        memory: 100Mi
-  redis:
-    replicas: 3
-    image: registry.suse.com/harbor/harbor-redis:2.1.1
-    customConfig:
-      - "dir /data"
-    securityContext:
-      runAsUser: 999
-      runAsGroup: 999
-      fsGroup: 999
-    resources:
-      requests:
-        cpu: 100m
-        memory: 100Mi
-      limits:
-        cpu: 400m
-        memory: 500Mi
-    storage:
-      keepAfterDeletion: true
-      persistentVolumeClaim:
-        metadata:
-          name: harbor-redis-data
-        spec:
-          accessModes:
-            - ReadWriteOnce
-          resources:
-            requests:
-              storage: 10Gi
-  auth:
-    secretPath: redis-auth
-----
-+
-At a minimum, the configuration options below should be customized as desired.
-For more configuration options, the link:https://github.com/spotahome/redis-operator#usage[Redis Operator Documentation] may be used.
-+
---
-[lowerroman]
-. Number of replicas for sentinel and Redis. Use a number higher than 2 for a highly-available installation
-. The storage size for the `persistentVolumeClaim`
---
+redisfailover:
+  enabled: true
+  name: harbor-redisfailover
+----
+
+... Configure {spr} to be connected to the external Redis
 +
-The `RedisFailover` configuration can be deployed using `kubectl`.
-For example, assuming `redis.yaml` is the file containing the `RedisFailover` object properties, the command would be:
+Extend the `harbor-values.yaml` file with the configuration specified below.
 +
-[source,bash]
+.harbor-values.yaml
+[source,yaml]
 ----
-kubectl create -n registry -f redis.yaml
+redis:
+  type: external
+  external:
+    addr: rfs-harbor-redisfailover:26379
+    sentinelMasterSet: mymaster // <1>
 ----
-... Wait until all the objects are prepared, specifically `rfs-harbor-redis`, the Sentinel deployment:
+<1> `mymaster` is a predefined value of redisfailover deployment and cannot be changed.
+
+... (Optional) Set up own password
++
+By default, if no secret and password is provided, the {spr} Helm chart will protect the deployed redis with a generated password. Custom password can also be provided:
```

Comment:
```suggestion
By default, if no secret and password are provided, the {spr} Helm chart will protect the deployed redis with a generated password. A custom password can also be provided:
```

Reply:
What does a "generated password" mean here? What/who generates it?

Reply:
thanks, i have changed this line